storage/kafka/src/main/java/io/apicurio/registry/kafka/KafkaRegistryStorage.java

```diff
@@ -48,102 +63,405 @@
 @ApplicationScoped
 public class KafkaRegistryStorage extends SimpleMapRegistryStorage implements KafkaRegistryStorageHandle {
 
+    private static final Logger log = LoggerFactory.getLogger(KafkaRegistryStorage.class);
+
+    @ConfigProperty(name = "registry.kafka.snapshot.requests", defaultValue = "1000")
+    long snapshotRequests;
+
+    @ConfigProperty(name = "registry.kafka.snapshot.period", defaultValue = "1200") // 2 days
```

Comment:
In Kafka itself the convention is to include the unit in the config parameter name. That communicates what the unit is without having to refer to the documentation. It might be a good idea to adopt the same convention here.

Reply:
OK